pick_kernel/kernel.py

```diff
@@ -1,31 +1,24 @@
+import asyncio
+import base64
 import os
 import sys
+from binascii import hexlify
+from queue import Empty
 
 from tornado.ioloop import IOLoop
-
 import zmq
 from zmq.eventloop import ioloop
-
-import base64
-
-from binascii import hexlify
-import os
-
-from queue import Empty
-
-ioloop.install()
 # Rely on Socket subclass that returns Futures for recv*
 from zmq.eventloop.future import Context
 
-from jupyter_client import AsyncKernelManager
-from jupyter_client.session import extract_header
+from ipykernel.jsonutil import json_clean
 from ipykernel.kernelbase import Kernel
 from ipykernel.kernelapp import IPKernelApp
+from jupyter_client import AsyncKernelManager
+from jupyter_client.session import extract_header
```

Comment:
Thank you for all the cleanup!

Reply:
No problem. Did it while watching a movie with Jake.